partiql-plan/src/main/java/org/partiql/plan/Version.java

```diff
@@ -23,7 +23,7 @@ public String name() throws UnsupportedCodeException {
         throw new UnsupportedCodeException(code);
     }
 
-    public static final int V1 = 1;
+    public static final int V1 = 0;
```

Comment:
(self-review) renumber to start with 1. Similarly for other enums where I got rid of the `UNKNOWN` enum variant and renumbered.